Update kinematic-chain-config.md (#1520)
viamrobotics/docs: docs/internals/kinematic-chain-config.md

{{% alert title="Info" color="info" %}}

> These reference frames are ingested by the Frame System, but not exposed through gRPC calls except for arms.

Reviewer: ```suggestion
These reference frames are ingested by the Frame System.
They are not exposed in the [client SDKs](/program/), with one exception.
```
[opt] if this is accurate

> For arms, you can use the GetKinematics() call to access kinematics information of the arm.

Reviewer: ```suggestion
If your resource is an [arm component](/components/arm/), you can use the [Kinematics()]((/components/arm/#kinematics)) method to access its kinematics information.
```

{{% alert title="Info" color="info" %}}

These reference frames are ingested by the Frame System.
They are not exposed in the [client SDKs](/program/), with one exception.

> If your resource is an [arm component](/components/arm/), you can use the [Kinematics()]((/components/arm/#kinematics)) method to access its kinematics information.

Reviewer: ```suggestion
If your resource is an [arm component](/components/arm/), you can use the [Kinematics()](/components/arm/#kinematics) method to access its kinematics information.
```
